code/WorkInProgress/Cael_Aislinn/ShieldGen/shield_capacitor.dm

```diff
@@ -3,7 +3,7 @@
 //pulls energy out of a power net and charges an adjacent generator
 
 /obj/machinery/shield_capacitor
-	name = "starscreen shield capacitor"
+	name = "Starscreen shield capacitor"
```

Comment:
\improper makes it interpret it as a lowercase name, so in instances of \the [src], it'd go the Starscreen shield capacitor